csharp_dotnetcore/09.Message-Routing/BotConfiguration.bot

```diff
@@ -1,10 +1,11 @@
 {
```

Comment:
These samples need to move from \botbuilder-samples\csharp_dotnetcore to \botbuilder-samples\samples\csharp_dotnetcore.